app/src/main/java/com/example/epamandroid/MainActivity.kt

```diff
@@ -1,12 +1,88 @@
 package com.example.epamandroid
 
+import android.graphics.Color
+import android.graphics.Color.*
+import android.graphics.PorterDuff
 import android.support.v7.app.AppCompatActivity
 import android.os.Bundle
+import android.support.design.widget.NavigationView
+import android.support.v4.view.GravityCompat
+import android.view.MenuItem
+import android.widget.ImageView
+import kotlinx.android.synthetic.main.activity_main.activity_main_toolbar
+import kotlinx.android.synthetic.main.activity_main.activity_main_drawer_layout
+import kotlinx.android.synthetic.main.activity_main.activity_main_nav_view
+
+class MainActivity : AppCompatActivity(), NavigationView.OnNavigationItemSelectedListener {
+
+    private val colorList: MutableList<String> = ArrayList()
 
-class MainActivity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+        configureSupportActionBar()
+        setBackgroundList()
+
+        val navigationHeaderIconButton = activity_main_nav_view
```

Comment:
@egorikftp How can I change the color of the icon better?

Reply:
You don't have a compound view, in this case, you change colour in a strange way 🙂